amethyst_assets/src/storage.rs

```diff
@@ -148,6 +148,8 @@ impl<A: Asset> AssetStorage<A> {
                     name,
                     tracker,
                 } => {
+
```

Comment:
Please remove the whitespace and squash the commits, then this is ready to get merged :+1: